apps/zotonic_mod_zotonic_site_management/priv/skel/blog/src/SITE_app.erl

```diff
@@ -0,0 +1,29 @@
+%% @author %%FULLNAME%%
+%% @copyright %%YEAR%% %%FULLNAME%%
+%% Generated on %%DATE%%
+%% @doc %%SITE%% application.
+
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%%     http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+
+-module(%%SITE%%_app).
+-author("%%FULLNAME%%").
+
+-behaviour(application).
+
+-export([ start/2, stop/1 ]).
+
+start(_StartType, _StartArgs) ->
```

Comment:
I suppose it need to either add them everywhere or get rid of them.

Reply:
You mean the specs?

Reply:
Oh.. I miss this word :facepalm:. Yes, `spec`'s.